src/app/work-item/work-item-list/work-item-detail/work-item-link/work-item-link.component.html

```diff
@@ -1,25 +1,37 @@
 <div class="link-container" id="wi-link">
     <div class="link-header" (click)="toggleLinkComponent()">
         <span class="content__title pull-left">
-            <!-- Refactoring needed -->
             <i *ngIf="showLinkComponent" class="fa fa-angle-down"></i>
             <i *ngIf="!showLinkComponent" class="fa fa-angle-up"></i>
         </span>
-        <div class="link-header-nav"><span id="wi-link-total">{{totalLinks}}</span><span class="link-header-nav-text">Total Links</span></div>
-        <div class="link-header-nav" *ngFor="let linkType of linkTypes">
-            <span>{{ linksGroupCount[linkType['id']] > 0 ? linksGroupCount[linkType['id']] : 0 }}</span><span class="link-header-nav-text">{{linkType.attributes['name']}}</span>
+        <div class="link-header-nav">
+          <span id="wi-link-total">
+            {{workItem.relationalData.totalLinkCount}}
+          </span>
+          <span class="link-header-nav-text">
+            Total Links
+          </span>
+        </div>
+        <div class="link-header-nav" *ngFor="let linkType of workItem.relationalData.linkDicts">
+            <span>
+              {{linkType.count}}
```

Comment:
@sanbornsen , Can you add id's for link types ,as you have added for total count. That would be great!